include/zephyr/drivers/flash/ra_flash_api_extensions.h

```diff
@@ -13,6 +13,11 @@ enum ra_ex_ops {
 	FLASH_RA_EX_OP_WRITE_PROTECT = FLASH_EX_OP_VENDOR_BASE,
 };
 
+enum qspi_flash_ex_op_types {
+	QSPI_FLASH_EX_OP_RESET = 0,
+	QSPI_FLASH_EX_OP_EXIT_QPI = 1,
```

Comment:
This define is in the same range as system wide definition for non-vendor space, and as such is not allowed.
Please refer to https://github.com/zephyrproject-rtos/zephyr/blob/03959a20f7834ffe50a51df2b2e45ef87d1b610b/include/zephyr/drivers/flash.h#L617-L631
for instruction on how to define vendor specific operation
and to https://github.com/zephyrproject-rtos/zephyr/blob/03959a20f7834ffe50a51df2b2e45ef87d1b610b/include/zephyr/drivers/flash/stm32_flash_api_extensions.h for an example of such definition.

Reply:
I think that there is also typo and QPI should be QSPI?

Reply:
We updated for "QSPI_FLASH_EX_OP_EXIT_QPI" as your advice.
And "QPI" isn't typo. We mentioned QPI mode (4-4-4).